lib/ts_utils/utils.py

```diff
@@ -14,7 +14,7 @@
 from packaging.requirements import Requirement
 
 try:
-    from termcolor import colored as colored  # pyright: ignore[reportAssignmentType]
+    from termcolor import colored as colored  # pyright: ignore[reportAssignmentType] # noqa: PLC0414
```

Comment:
We should probably just disable this rule, I don't think it's that helpful in typed code where "import X as X" is meaningful.

Reply:
Some context: This rule is disabled in `*.pyi` files (https://github.com/astral-sh/ruff/pull/3761) and there's talks of disabling it oiut of the box for `__init__.py` (or it can be easily configured to do so https://github.com/astral-sh/ruff/issues/6294#issuecomment-2018779663).
So it only flags possible accidental re-exports from non-index/init runtime modules.

That being said, we're not a (public) library either (technically ts_utils this is a local lib), and I believe typeshed maintainers should be able to quickly spot an involuntary reexports.